vue/configurator/src/views/ConnectView.vue

```diff
@@ -134,6 +134,35 @@ export default class ConnectView extends Vue {
                 value: "",
                 defaultValue: "",
               },
+              "data-widget-theme": {
```

Comment:
can you confirm that there is actually a `data-` prop of this name?

Reply:
There wasn't a prop of this name, but my idea was that I wanted it to somehow be integrated with the configurator rather than the app.vue file. 
An alternative way I was thinking about it so I wouldn't create this new prop would be to add a property "theme" in the WidgetDefinition like this. But I thought that might be a more impactful structure change than adding a new data- prop. What do you think? Is there a better way to handle this inside the widgets?
```
export interface WidgetDefinition {
  name: string;
  props: { [key: string]: WidgetPropDefinition };
  dataWidget?: string;
  theme?: string;
  javascript: string;
  css: string;
}
```

Reply:
since there isn't a prop of this name in the docs, it would be confusing for end users to see this scheme option as a data prop. adding it to `WidgetDefinition` is a better idea (basically, your drop down will affect the different widgets individually).

another option to pursue would be adding a custom input field in the configurator file that affects the previews altogether.

Reply:
agreed! updated!